login.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/630f3095fb.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://practical-poitras-d410a0.netlify.app/packages/index.css">
+    <link href="https://fonts.googleapis.com/css2?family=Quintessential&family=Roboto:ital,wght@0,300;1,400&family=Zen+Kurenaido&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <nav class="navbar">
+        <div class="navbar-title">
+            Astra
+        </div>
+        <div>
+            <a href="index.html" class="navbar-link mr-s">Home</a>
+            <a href="product.html" class="navbar-link">product</a>
+        </div>
+        <div class="navbar-items">
+            <div>
+                <ul class="navbar-list">
+                    <li class="navbar-item navbar-icon">
+                        <a href="cart.html" class="navbar-link">
+                            <span class="btn-icon"><i class="fas fa-shopping-cart"></i>
+                            </span>
+                        </a>
+                    </li>
+                    <li class="navbar-item navbar-icon">
+                        <a href="wishlist.html" class="navbar-link">
+                            <i class="fas fa-heart"></i></a>
+                    </li>
+                </ul>
+            </div>
+            <a href="login.html" class="btn btn-primary">Login</a>
+        </div>
+    </nav>
+    <div class="container">
+        <div class="form-box">
+            <div class="form form-primary">
+                <div class="form-control ">
+                    <label for="username" class="input-label">Username</label>
+                    <input type="text" placeholder="Enter your name" class="input">
+                </div>
+                <div class="form-control">
+                    <label for="username" class="input-label">password</label>
+                    <input type="text" placeholder="Enter your password" class="input">
```

Comment:
While using input and label, you should have id attribute in input tag which is equal to the label's for attribute. For example, here for="username" in the label will match input's id="username". This is a good practice to incorporate.